asapdiscovery-modeling/asapdiscovery/modeling/modeling.py

```diff
@@ -577,19 +577,24 @@ def split_openeye_mol(
         Molecule filter object that contains the filter criteria.
     prot_cutoff_len : int, default=10
         Minimum number of residues in a protein chain required in order to keep
+    keep_one_lig : bool, default=True
+        Only keep one copy of the ligand in lig_mol
 
     Returns
     -------
-    namedtuple
-        A namedtuple containing the protein, ligand, and water molecules.
+    dict
+        A dict containing the protein ("prot"), ligand ("lig"), water atoms ("wat"),
+        and all other atoms ("oth")
     """
     # These are the objects that will be returned at the end
     lig_mol = oechem.OEGraphMol()
```

Comment:
These are all 2D right? perhaps at some point we will need to preserve the 3D information. Lets think about how to address this best. Perhaps its as simple as changing to `OEMol`

Reply:
the `OEGraphMol` class does have 3D coordinates. it's just a specific `OEBaseMol` subclass that's used for single-conformation molecules. all information for each atom should be preserved from the original complex molecule to the split molecules